src/assets/translations/en/main.json

```diff
@@ -604,9 +600,23 @@
         "failed": "There was an issue with your deposit."
       }
     },
+    "stake": {
+      "amountToStake": "Amount To Stake",
+      "stakeYour": "Stake your %{opportunity}",
+      "estimatedReturns": "Estimated Yearly Returns*",
+      "estimateDisclaimer": "*Estimates fluctuate with market conditions.",
+      "footerDisclaimer": "By continuing you agree to our ",
+      "footerDisclaimerLink": "terms of service.",
+      "status": {
+        "pending": "Your transaction has been broadcast and is pending confirmation.",
+        "success": "You have successfully staked your %{opportunity}.",
+        "failed": "There was an issue with your stake."
```

Comment:
Request change to 'There was an issue with your staking transaction.'

Reply:
Done in b6502b6c3a5ebb85299da6fcab2ca3491697b7ea